packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -282,57 +392,516 @@ function LivePreviewDemo(props: Args): JSX.Element {
     );
   }
 
-  // loop through all PSEUDO_STATES + create headers for HOVERABLE_OPTIONS, PSEUDO_STATES
-  function TagState({
+  // // loop through all PSEUDO_STATES + create headers for HOVERABLE_OPTIONS, PSEUDO_STATES
```

Comment:
I left in the commented-out parts of the 3 x screenshot-related stories for context: this is the loop that is now externalized and reused by all three, so this code doesn't need to exist redundantly for each (But I left it commented out for context for now)